src/Utils.js

```diff
@@ -12,6 +12,8 @@ import {
   REACT15,
 } from './version';
 
+export const ITERATOR_SYMBOL = typeof Symbol === 'function' && Symbol.iterator;
```

Comment:
Yes, that's true, because Symbols can't be polyfilled (because of `typeof`) and `Symbol.for` - ie, it'd not be a shim, but a sham.

I suppose I don't need to make it a sticking point here, but I'd prefer a more robust test than truthiness - for now this is fine, but I'll extract https://github.com/ljharb/object.assign/blob/master/hasSymbols.js out to a module or two and we can use that instead :-)